packages/angular-virtual/e2e/app/src/app/app.config.ts

```
@@ -0,0 +1,7 @@
+import type { ApplicationConfig } from '@angular/core'
```

ESLint

[error] `@angular/core` type import should occur after import of `./app.routes`

(import/order)